### ISSUE TITLE
Remove mockito 

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/dodal#670]
We originally started with using `mockito` as a mocking framework. However, it is now only used in a couple of places and gives us nothing over `pytest`/`unittest` so we should remove it to standardise and reduce dependencies.

## Acceptance Criteria
* `mockito` is removed as a dependency
* Tests all still pass

**Comment by Relm-Arrowny** [comment]
can I pick this issue up, I got curious of what mockito is so I toy around with it and in the processes almost fixed this issue?

**Comment by DominicOram** [comment]
Yes, please do. Just assign yourself so we know you're working on it
